src/stores/useGovStore.ts

```diff
@@ -34,12 +38,29 @@ export const useGovStore = defineStore('govStore', {
           await this.blockchain.rpc?.getGovProposals(status)
         );
         if (status === '2') {
-          proposals?.proposals?.forEach(async (x1) => {
-            await this.fetchTally(x1.proposal_id).then((res) => {
-              x1.final_tally_result = res?.tally;
+          proposals?.proposals?.forEach(async (item) => {
+            await this.fetchTally(item.proposal_id).then((res) => {
+              item.final_tally_result = res?.tally;
+              // 
+             
             });
+            if (this.walletstore.connectedWallet?.cosmosAddress) {
+              try {
+                await this.fetchProposalVotesVoter(item.proposal_id,this.walletstore.currentAddress).then((res) => {
```

Comment:
`await` will cause delay for initializing the page.

please separately load  and computed the proposals and votes